docs/source/core_concepts/glossary.rst

```diff
@@ -300,6 +295,13 @@ Ordered batch preserves only the sequence of transactions applying to a ledger.
 All the transactions that able to pass stateful validation within a batch will be applied to a ledger.
 Validation failure of one transaction would NOT directly imply the failure of the whole batch.
 
+Multisignature Transactions
+===========================
+
+A transaction which has the `quorum`_ greater than one is considered as multisignature (also called mst).
+For its `stateful validity <#stateful-validation>`__ it is required to be confirmed by the signatories of the creator account.
```

Comment:
It is required to be signed by a number of public keys of transactions creator (the number is equal to quorum)

Reply:
Otherwise it is too complex

Reply:
In fact `signatories` might differ from the `public keys of transactions creator`. Might be just add a `Signatory` chapter and reference it?